listenbrainz/webserver/views/index.py

```diff
@@ -103,11 +106,26 @@ def current_status():
     except DatabaseException as e:
         user_count = 'Unknown'
 
+    listen_counts_per_day: List[dict] = []
+    for delta in range(2):
+        try:
+            day = datetime.utcnow() - relativedelta(days=delta)
+            day_listen_count = _redis.get_listen_count_for_day(day)
+        except:
+            current_app.logger.error("Could not get today's listen count from redis", exc_info=True)
```

Comment:
This error message could be wrong half of the time. Maybe give the date of the day that can't be fetched instead of saying "today's"?

Reply:
whoops, missed that.